tests/end2end/test_insert_mode.py

```diff
@@ -46,7 +46,7 @@ def test_insert_mode(file_name, source, input_text, auto_insert, quteproc):
         quteproc.press_keys(input_text)
     elif source == 'clipboard':
         quteproc.send_cmd(':debug-set-fake-clipboard "{}"'.format(input_text))
-        quteproc.send_cmd(':paste-primary')
+        quteproc.send_cmd(':insert-text {clipboard}')
```

Comment:
Is this `{clipboard}` or `{primary}`, as the deprecation message for `:paste-primary` says?